### PR TITLE
pkg/oc/cli/admin/release/new: Document SRC=DST positional args

### PR DESCRIPTION
**Pull request by wking**
Make this important functionality more discoverable, since it's very useful for testing operators ;).

CC @smarterclayton, @sallyom, @sjenning

### CODE REVIEW

File: pkg/oc/cli/admin/release/new.go [hunk 1]
```diff
@@ -47,12 +47,12 @@ func NewNewOptions(streams genericclioptions.IOStreams) *NewOptions {
...
 	cmd := &cobra.Command{
-		Use:   "new",
+		Use:   "new [SRC=DST ...]",
 		Short: "Create a new OpenShift release",
 		Long: templates.LongDesc(`
 			OpenShift uses long-running active management processes called "operators" to
-			keep the cluster running and manage component lifecycle. This command assists
-			composing a set of images and operator definitions into a single update payload
+			keep the cluster running and manage component lifecycle. This command
+			composes a set of images and operator definitions into a single update payload
```
Comment (wking) [inline pkg/oc/cli/admin/release/new.go]: I can pull this orthogonal change out into its own PR if that helps with review.

Comment (smarterclayton) [reply pkg/oc/cli/admin/release/new.go]: It’s fine

**Comment by sjenning** [comment]
thanks for this @wking! :tada:

**Pull request merged**